src/__internal__/form-field/form-field.component.tsx

```diff
@@ -90,6 +99,8 @@ const FormField = ({
   validationIconId,
   ...rest
 }: FormFieldProps) => {
+  const { validationRedesignOptIn } = useContext(NewValidationContext);
```

Comment:
Shall we move this context to the `Label` component instead? I don't think `FormField` needs to care about the `validationRedesignOptIn` and we can add it as a guard to https://github.com/Sage/carbon/blob/19da80816cc29da9a6c4b3b47412f967aeff99b5/src/__internal__/label/label.component.tsx#L100

Reply:
Good idea @edleeks87 👍🏼

Reply:
@edleeks87 I noticed if we do move the context into `Label` and bypass `FormField`, we would need to move the `invalidValidationProp` check into each input component as well. I have experimented creating a new helper for this, but to avoid delaying this fix, I will leave it as it is and come back to it in the future.

Reply:
I'm not sure that's the case tbh but I'm happy to leave it as it's only one level we're trying to bypass and we'll eventually be removing it all anyway 👍